frappe/website/doctype/website_theme/website_theme.py

```diff
@@ -111,7 +111,7 @@ def generate_bootstrap_theme(self):
 		content = content.replace("\n", "\\n")
 		command = ["node", "generate_bootstrap_theme.js", output_path, content]
 
-		process = Popen(command, cwd=frappe.get_app_path("frappe", ".."), stdout=PIPE, stderr=PIPE)
+		process = Popen(command, cwd=frappe.get_app_source_path("frappe"), stdout=PIPE, stderr=PIPE)
```

Comment:
Notably, this is a runtime asset generator discoverable in such a way and which can't work in its current implementation from a regular python module.